### ISSUE TITLE
Importing ABCs from "collections" is deprecated (will break in 3.8)

### ISSUE THREAD

**Issue by tvst** [gorakhargosh/watchdog#579]
Abstract base classes should be imported from `collections.abc` from now on. See this warning my pytests are throwing:

```
(...)/lib/python3.7/site-packages/watchdog/utils/bricks.py:175: DeprecationWarning:

Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
```

**Comment by rrzaripov** [comment]
@tvst thank you for reporting, this is will be fixed in next release, see #462

**Comment by tvst** [comment]
Awesome. Sorry for the noise! Github didn't show me any good matches when I searched :confused:
